v2/pkg/operators/extractors/compile.go

```diff
@@ -10,13 +10,13 @@ import (
 
 // CompileExtractors performs the initial setup operation on an extractor
 func (e *Extractor) CompileExtractors() error {
-	var ok bool
+	//var ok bool
```

Comment:
Could you please remove the commented code?

Reply:
Done